src/main/Test/java/mate/academy/security/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy.security;
+
+import static org.mockito.ArgumentMatchers.any;
+
+import java.util.Optional;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import mate.academy.service.RoleService;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+class AuthenticationServiceTest {
```

Comment:
Think about what other negative tests you can add. At what inputs will we get errors?